package.json

```diff
@@ -7,10 +7,12 @@
     "test": "tests"
   },
   "engines": {
-    "node": "~4.2.0"
+    "node": "^4.2.0"
```

Comment:
Ah yep, we've done that in the `integrate-react` branch already, but never merged it back.